### PR TITLE
Multinomial bayes

### PR DESCRIPTION
**Pull request by JeffersonH44**
@targos @maasencioh

### CODE REVIEW

**Comment by JeffersonH44** [comment]
@targos @maasencioh

**Comment by maasencioh** [comment]
the test takes a very long time

**Comment by JeffersonH44** [comment]
sorry for the long pr, and tests for me is not that long.
deepinscreenshot_select-area_20170905165945

**Comment by maasencioh** [comment]
at the begining it took more time

capture d ecran 2017-09-05 a 10 03 54

**Comment by targos** [comment]
You have to remove the coverage report and put `coverage` in .gitignore

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,274 @@
+'use strict';
```
Comment (targos) [inline lib/index.js]: this file should also be removed and put in the gitignore

File: package.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
   "name": "ml-naivebayes",
-  "version": "2.0.0",
+  "version": "1.0.3",
   "description": "Naive bayes library",
-  "main": "src/index.js",
-  "directories": {
-    "lib": "src",
-    "test": "test"
-  },
+  "main": "lib/index.js",
```
Comment (targos) [inline package.json]: can you add `"module": "src/index.js"` ?

**Comment by JeffersonH44** [comment]
ok, it's done @targos

**Comment by targos** [comment]
thank you!

**Comment by targos** [comment]
this is a breaking change. do a major version if you release it

**Pull request merged**